Support for Prettier version 3

**Is your feature request related to a problem?**
Cannot currently be used with version 3 of Prettier

```
npm i -D @ianvs/prettier-plugin-sort-imports
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: stripe-connect-app-embed@1.0.0
npm ERR! Found: prettier@3.0.0
npm ERR! node_modules/prettier
npm ERR!   dev prettier@"^3.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer prettier@"2.x" from @ianvs/prettier-plugin-sort-imports@4.0.2
npm ERR! node_modules/@ianvs/prettier-plugin-sort-imports
npm ERR!   dev @ianvs/prettier-plugin-sort-imports@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/tvanantwerp/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tvanantwerp/.npm/_logs/2023-07-10T15_00_10_576Z-debug-0.log
```

**Describe the solution you'd like**
Support for version 3 of Prettier.

[Related issue](https://github.com/trivago/prettier-plugin-sort-imports/issues/240) in `@trivago/prettier-plugin-sort-imports`

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
     },
     "peerDependencies": {
         "@vue/compiler-sfc": ">=3.0.0",
-        "prettier": "2.x"
+        "prettier": "2 || 3"
     },
     "peerDependenciesMeta": {
         "@vue/compiler-sfc": {

